object-handlers.go

```diff
@@ -946,21 +946,40 @@ func (api objectAPIHandlers) CompleteMultipartUploadHandler(w http.ResponseWrite
 		completeParts = append(completeParts, part)
 	}
 	// Complete multipart upload.
+	// Send 200 OK
+	setCommonHeaders(w)
+	w.WriteHeader(http.StatusOK)
```

Comment:
Writing this status will cause a problem with the error response later, you cannot write HTTP status twice to HTTP response writer. 

Once you write a test case you will see the problem reproducing an error condition.